### PR TITLE
docs(core): polishes README for recency & clearer routing 

### PR DESCRIPTION
**Pull request by goodroot**
* general structural polish for readability
* updates sections to route to docs when appropriate
* removes some context to get into useful paths quicker
* refers to separate docs repo, routes via template to docs repo vs. issues here

### CODE REVIEW

**Comment by questdb-butler** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  Please update the PR title to match this format:
`type(subType): description`

Where `type` is one of:
`feat`, `fix`, `chore`, `docs`, `style`, `refactor`, `perf`, `test`, `ci`, `revert`

And: `subType` is one of:
`build`, `sql`, `log`, `mig`, `core`, `ilp`, `pgwire`, `http`, `conf`, `ui`, `wal`

For Example:

```
perf(sql): improve pattern matching performance for SELECT sub-queries
```
  </td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 9b5ed4219644b5dddea231b18ba5279b1cb0738e
</p>

**Comment by goodroot** [comment]
Build error appears un-releated

File: README.md [hunk 1]
```diff
@@ -12,43 +12,89 @@
...
 </div>
 
+<p>&nbsp;</p>
+
+## Benefits of QuestDB
+
+QuestDB excels with:
+
+- financial market data
+- IoT sensor
```
Comment (tris0laris) [inline README.md:57]: Should we change to:
IoT sensors or IoT sensor data

File: README.md [hunk 2]
```diff
@@ -12,43 +12,89 @@
...
+- real-time dashboards
+- time-series data with high data cardinality
+
+Feature highlights include:
+
+- SQL with powerful time-series extensions: PGWire, HTTP API
+- High-speed ingestion through the InfluxDB Line Protocol over TCP or HTTP
```
Comment (tris0laris) [inline README.md:64]: I would remove this bit of complexity since the default path is HTTP: "over TCP or HTTP"

File: README.md [hunk 3]
```diff
@@ -57,96 +103,138 @@ Our [live demo](https://demo.questdb.io/) is provisioned with the latest QuestDB
...
-[QuestDB Enterprise](https://questdb.io/enterprise/) (self-hosted or BYOC) allows your organization to run the database at scale, securely.
-Additional features include multi-primary ingestion, read replica(s), cold storage integration, Role-based access control,
-monitoring, TLS, and more.
+- [.NET](https://questdb.io/docs/clients/ingest-dotnet/)
+- [C/C++](https://questdb.io/docs/clients/ingest-c-and-cpp/)
+- [Go](https://questdb.io/docs/clients/ingest-go/)
+- [Java](https://questdb.io/docs/clients/java_ilp/)
+- [NodeJS](https://questdb.io/docs/clients/ingest-node/)
+- [Python](https://questdb.io/docs/clients/ingest-python/)
+- [Rust](https://questdb.io/docs/clients/ingest-rust/)
```
Comment (tris0laris) [inline README.md:139]: Could we re-order With Python first?

File: README.md [hunk 4]
```diff
@@ -12,43 +12,89 @@
...
+- financial market data
+- IoT sensor
+- real-time dashboards
+- time-series data with high data cardinality
+
+Feature highlights include:
+
+- SQL with powerful time-series extensions: PGWire, HTTP API
+- High-speed ingestion through the InfluxDB Line Protocol over TCP or HTTP
+- Strong and efficient performance on limited hardware (Raspberry Pi, 4 workers)
```
Comment (tris0laris) [inline README.md:65]: I would remove examples in brackets

File: README.md [hunk 5]
```diff
@@ -12,43 +12,89 @@
...
+
 ## Try QuestDB
 
-Our [live demo](https://demo.questdb.io/) is provisioned with the latest QuestDB release and sample datasets:
+The [live, public demo](https://demo.questdb.io/) is provisioned with the latest
+QuestDB release and sample datasets:
 
 - Trips: 10 years of NYC taxi trips with 1.6 billion rows
 - Trades: live crypto market data with 30M+ rows per month
 - Pos: geolocations of 250k unique ships over time
```
Comment (tris0laris) [inline README.md:91]: Can we add Crypto dashboard example too?

File: README.md [hunk 6]
```diff
@@ -12,43 +12,89 @@
...
+  [high data cardinality](https://questdb.io/glossary/high-cardinality/)
+
+And why use a time-series database?
+
+Beyond performance and efficiency, with a specialized [time-series database](https://questdb.io/glossary/time-series-database/), you don't need to worry about:
+
+- out-of-order data
+- duplicates
+- exactly one semantics
+- frequency of ingestion
```
Comment (tris0laris) [inline README.md:81]: I would change "frequency of ingestion". Themes we can explore that are interesting:
streaming data (low latency)
dealing with a large number of concurrent requests
Minimal time between data ingestion and ability to read the data

File: README.md [hunk 7]
```diff
@@ -12,43 +12,89 @@
...
+- many other expensive details found in demanding real-world scenarios
```
Comment (tris0laris) [inline README.md:82]: i would remove because too vague

File: README.md [hunk 8]
```diff
@@ -57,96 +103,138 @@ Our [live demo](https://demo.questdb.io/) is provisioned with the latest QuestDB
...
+- role-based access control
+- TLS encryption
+- support SLAs, enhanced monitoring and more
 
-Checkout [our benchmark blog post](https://questdb.io/blog/2024/02/26/questdb-versus-influxdb/)
-which compares QuestDB and InfluxDB across functionality, maturity and performance.
+Visit the [Enterprise page](https://questdb.io/enterprise/) for further details
+and contact information.
+
+### QuestDB performance vs. other open-source TSDBs
```
Comment (tris0laris) [inline README.md:192]: MongoDB is not a TSDB, I would change the title slightly to reflect that

File: README.md [hunk 9]
```diff
@@ -12,43 +12,89 @@
...
+<p>&nbsp;</p>
 
 <div align="center">
-  <a href="https://demo.questdb.io">
-    <img alt="QuestDB Web Console showing a SQL statement and query result" src="https://raw.githubusercontent.com/questdb/questdb/master/.github/console.png" width="600" />
+  <a href="https://demo.questdb.io/">
+    <img alt="QuestDB Web Console showing a SQL statement and query result" src="https://raw.githubusercontent.com/questdb/questdb/master/.github/console.png" width="900" />
   </a>
+  <p><em>QuestDB Web Console</em></p>
 </div>
```
Comment (tris0laris) [inline README.md:49]: Comment on the intro paragraph:

I like the refocus on simplicity, but also losing the focus on "obsession for performance" that I think should be maintained

File: README.md [hunk 10]
```diff
@@ -12,43 +12,89 @@
...
-ingestion and fast SQL queries with operational simplicity.
+---
 
-QuestDB is well-suited for financial market data, IoT sensor data and real-time dashboards. It shines for datasets with [high cardinality](https://questdb.io/glossary/high-cardinality/)
-and is a drop-in replacement for InfluxDB via support for the InfluxDB Line Protocol.
+QuestDB is the fastest growing open-source time-series database.
 
-QuestDB implements ANSI SQL with native time-series SQL extensions. These SQL extensions make it simple to filter and downsample data,
-or correlate data from multiple sources using relational and time-series joins.
+It provides **high throughput ingestion** and **fast SQL queries**.
```
Comment (tris0laris) [inline README.md:34]: lacks a bit of punch. any suggestions?

**Review by tris0laris — DISMISSED** [review]

**Review by tris0laris — APPROVED** [review]

**Review by tris0laris — DISMISSED** [review]

**Review by tris0laris — DISMISSED** [review]

File: README.md [hunk 11]
```diff
@@ -12,43 +12,100 @@
...
+The [live, public demo](https://demo.questdb.io/) is provisioned with the latest
+QuestDB release and sample datasets:
 
 - Trips: 10 years of NYC taxi trips with 1.6 billion rows
 - Trades: live crypto market data with 30M+ rows per month
 - Pos: geolocations of 250k unique ships over time
 
+Use example queries or write your own!
+
+_The public demo queries over 1.6BN rows and uses 24 cores out
```
Comment (puzpuzpuz) [inline README.md:106]: We're using 46 shared workers, so it's more precise to say that we use 48 cores.
```suggestion
_The public demo queries over 1.6BN rows and uses 48 cores out
```

File: README.md [hunk 12]
```diff
@@ -12,43 +12,100 @@
...
+of 96 cores on a `c5.metal` instance._
```
Comment (puzpuzpuz) [inline README.md:107]: @sklarsa are we still using `c5.metal`? I recall that it's a different instance type now.

**Review by puzpuzpuz — DISMISSED** [review]

File: README.md (hunk 12 above)
Comment (sklarsa) [reply README.md:107]: `r6a.12xlarge`: 48 cpu/348GB

File: README.md [hunk 13]
```diff
@@ -12,43 +12,100 @@
...
+QuestDB release and sample datasets:
 
 - Trips: 10 years of NYC taxi trips with 1.6 billion rows
 - Trades: live crypto market data with 30M+ rows per month
 - Pos: geolocations of 250k unique ships over time
 
+Use example queries or write your own!
+
+_The public demo queries over 1.6BN rows and uses 48 cores out
+of 96 cores on a `c5.metal` instance._
```
Comment (goodroot) [inline README.md:107]:
```suggestion
_The public demo queries over 1.6BN rows and uses 48 cores out
of 96 cores on a `r6a.12xlarge` 348GB instance._
```

File: README.md (hunk 12 above)
Comment (goodroot) [reply README.md]: Cool. Updated!

I'm unable to merge. :) 

Might someone click the button?

File: README.md [hunk 14]
```diff
@@ -12,43 +12,100 @@
...
+The [live, public demo](https://demo.questdb.io/) is provisioned with the latest
+QuestDB release and sample datasets:
 
 - Trips: 10 years of NYC taxi trips with 1.6 billion rows
 - Trades: live crypto market data with 30M+ rows per month
 - Pos: geolocations of 250k unique ships over time
 
+Use example queries or write your own!
+
+_The public demo queries over 1.6BN rows and uses 48 cores out
```
Comment (puzpuzpuz) [inline README.md:106]:
```suggestion
_The public demo queries over 1.6BN rows and uses a `r6a.12xlarge` 48 vCPU and 348GB RAM instance._
```

File: README.md (hunk 12 above)
Comment (puzpuzpuz) [reply README.md]: Thanks Steve!

As for the merge permissions, only Alex and Vlad have them.

File: README.md [hunk 15]
```diff
@@ -12,43 +12,100 @@
...
+QuestDB release and sample datasets:
 
 - Trips: 10 years of NYC taxi trips with 1.6 billion rows
 - Trades: live crypto market data with 30M+ rows per month
 - Pos: geolocations of 250k unique ships over time
 
+Use example queries or write your own!
+
+_The public demo queries over 1.6BN rows and uses a `r6a.12xlarge` 48 vCPU and 348GB RAM instance._
+of 96 cores on a `r6a.12xlarge` 348GB instance._
```
Comment (puzpuzpuz) [inline README.md:107]: This line needs to be removed.

File: README.md (hunk 13 above)
Comment (puzpuzpuz) [reply README.md]: That's because QDB reserves 2 spare cores for OS.

File: README.md (hunk 15 above)
Comment (goodroot) [reply README.md:107]: Lol yes, on it.

**Review by puzpuzpuz — APPROVED** [review]

**Pull request merged**